WebDriverAgentLib/Commands/FBSessionCommands.m

```diff
@@ -43,6 +43,7 @@
 static NSString* const INCLUDE_NON_MODAL_ELEMENTS = @"includeNonModalElements";
 static NSString* const ACCEPT_ALERT_BUTTON_SELECTOR = @"acceptAlertButtonSelector";
 static NSString* const DISMISS_ALERT_BUTTON_SELECTOR = @"dismissAlertButtonSelector";
+static NSString* const AUTO_ADJUST_SCREENSHOT_ORIENTATION = @"autoAdjustScreenshotOrientation";
```

Comment:
I think we could make it more flexible for clients by explicitly providing the appropriate orientation value, like `UIImageOrientationLeft` or `UIImageOrientationUp` and leaving it to `auto` by default

Reply:
Good idea